src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExceptionListenerPass.php

```diff
@@ -27,8 +30,9 @@ public function process(ContainerBuilder $container)
             return;
         }
 
+        // to be removed in 5.0
         // register the exception controller only if Twig is enabled and required dependencies do exist
-        if (!class_exists('Symfony\Component\ErrorRenderer\Exception\FlattenException') || !interface_exists('Symfony\Component\EventDispatcher\EventSubscriberInterface')) {
+        if (!interface_exists(EventSubscriberInterface::class) || $container->hasDefinition('exception_listener')) {
             $container->removeDefinition('twig.exception_listener');
```

Comment:
I still need to check this again when both FrameworkBundle and TwigBundle are enabled, for the rest, ready for review.